cypress/support/commands.js

```diff
@@ -24,11 +24,23 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
+/**
+ * Runs necessary test setup to run a clean test.
+ */
 Cypress.Commands.add('testSetup', () => {
-  cy.request('/cypress_rails_reset_state');
-
   // Required for the moment because of https://github.com/cypress-io/cypress/issues/781
   cy.clearCookies();
+
+  cy.getCookies().then((cookie) => {
```

Comment:
I tested this change @aitchiss and @katiedavis and it seems to hold as per [my comment on the Cypress issue](https://github.com/cypress-io/cypress/issues/781#issuecomment-769361285) related to this.

Reply:
I'll pull and run through

Reply:
I'm just concerned it will be flakey again but I can't reproduce

Reply:
Yeah, I can definitely bump the time, I put it to `1ms` for the time being, but let me give it a bump. It still seems fine on CI though.

Reply:
It seems fixed on my end